src/Libraries/Analysis/PointData.cs

```diff
@@ -2,12 +2,14 @@
 using System.Collections.Generic;
 using System.Linq;
 using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
 
 namespace Analysis
 {
     /// <summary>
     /// A class for storing structure point analysis data.
     /// </summary>
+    [IsVisibleInDynamoLibrary(false)]
```

Comment:
These are hidden, but not removed from the library so that existing workspaces continue to function, but the nodes are not available in the library.